modules/ROOT/pages/core-operations.adoc

```diff
@@ -300,51 +300,74 @@ Using the append and prepend methods may invalidate an existing JSON document.
 You can use xref:subdocument-operations.adoc[sub-document operations] if you want to have true JSON-aware prepend and append operations which add values to JSON arrays.
 ====
 
-[source,python]
+[source,java]
 ----
-append(docid, fragment)
-prepend(docid, fragment)
+append(document);
+prepend(document);
```

Comment:
This needs a bit more, these samples don't make sense without the corresponding code to create the JsonDocument.

Reply:
I feel the code samples included as part of this section will be enough to explain `append` method. After that, `prepend` method example would be exactly similar. Let me know your thoughts on it.

Reply:
Also, compilation errors of the java snippets have been corrected.